msgraph-developer-proxy-plugins/Guidance/SdkGuidancePlugin.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Graph.DeveloperProxy.Abstractions;
+using System.Text.RegularExpressions;
+using Titanium.Web.Proxy.Http;
+
+namespace Microsoft.Graph.DeveloperProxy.Plugins.Guidance;
+
+public class SdkGuidancePlugin : IProxyPlugin {
```

Comment:
One more thought occurred to me: as we're supporting more than Graph, wouldn't it be clearer to name this plugin `GraphSdkGuidancePlugin`, in case we or folks will introduce additional plugins with guidance for other SDKs?